projects/plugins/jetpack/_inc/client/my-plan/my-plan-body.jsx

```diff
@@ -279,6 +279,37 @@ class MyPlanBody extends React.Component {
 			);
 		};
 
+		const getSocialCard = () => {
+			return (
+				<div className="jp-landing__plan-features-card">
+					<div className="jp-landing__plan-features-img">
+						<img
+							src={ imagePath + '/jetpack-search-icon.svg' }
+							className="jp-landing__plan-features-icon"
+							alt={ __( 'A Jetpack Site with the power of Jetpack Search', 'jetpack' ) }
+						/>
+					</div>
+					<div className="jp-landing__plan-features-text">
+						<h3 className="jp-landing__plan-features-title">
+							{ __( 'Instant search and filtering', 'jetpack' ) }
+						</h3>
+						<p>
+							{ __(
+								'Relevant search results and filtering tightly integrated with your theme.',
+								'jetpack'
+							) }
+						</p>
+						<Button
+							onClick={ this.handleButtonClickForTracking( 'view_search_customizer' ) }
+							href={ this.props.siteAdminUrl + 'admin.php?page=jetpack-search-configure' }
+						>
+							{ __( 'Customize Search', 'jetpack' ) }
+						</Button>
+					</div>
+				</div>
+			);
+		};
+
```

Comment:
This looks to be a copy of the Search card. We should update that. I'm not sure when this card is displayed, but we should test that it looks ok too.

Reply:
I will do that!